recirq/quantum_chess/pauli_decomposition.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Util functions used for decompose the given measurement matrix into PauliStrings.
```

Comment:
nit: Recommend "Utility functions used for decomposing ..." to avoid abbreviations and fix tense.

Reply:
Thanks Doug! It's fixed.